R/SupervisedTask.R

```diff
@@ -29,4 +29,6 @@ print.SupervisedTask = function(x, print.target = TRUE, print.weights = TRUE, ..
   if (print.weights)
     catf("Has weights: %s", td$has.weights)
   catf("Has blocking: %s", td$has.blocking)
+  if (print.target)
+    catf("Is spatial: %s", td$is.spatial)
```

Comment:
Why is this controlled by `print.target`? It's a property of the task, isn't it?